pytest_tests/lib/helpers/quota.py

```diff
@@ -65,3 +66,6 @@ def get_and_verify_container_quota(self, wallet: NodeWallet, cid: str, expected_
         assert quota_dict["hard"] == expected_hard, (
             f"Invalid hard quota, expected: {expected_hard}, got: {quota_dict['hard']}"
         )
+
+    def wait_until_quota_values_reported(self):
+        time.sleep(len(self.neofs_env.storage_nodes))
```

Comment:
Probably there is a way to check it via CLI. @carpawell?

Reply:
RPC is for FS chain's neo-go (part of the IR in our tests, if i remember correctly)

Reply:
> if polling is ok for us

It's better than `time.sleep()`.